src/ui/components/Navigation/Core/TableView/TableView.scss

```diff
@@ -249,6 +249,33 @@
         }
     }
 
+    &__edit-favorites-alias-btn {
+        visibility: hidden;
+        margin-right: 5px;
+        min-width: 28px;
+        min-height: 28px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        border-radius: 4px;
+
+        &_isVisible {
```

Comment:
`isVisible` -> `visible`
let's not use camelCase in css classes

Reply:
no problem, fixed